### PR TITLE
Add events on parameter changes

### PR DESCRIPTION
**Pull request by DimaStebaev**

### CODE REVIEW

**Review by yavrsky — APPROVED** [review]

File: proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol [hunk 1]
```diff
@@ -108,11 +108,39 @@ contract DepositBoxERC20 is DepositBox, IDepositBoxERC20 {
 
     event Escalated(uint256 id);
 
+    /**
+     * @dev Emitted when big transfer threshold is changed
+     */
+    event BigTransferThresholdIsChanged(
+        bytes32 schainHash,
```
Comment (payvint) [inline proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol]: Let's add indexed parameters

File: proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol [hunk 2]
```diff
@@ -267,7 +295,13 @@ contract DepositBoxERC20 is DepositBox, IDepositBoxERC20 {
         onlySchainOwner(schainName)
     {
         bytes32 schainHash = _schainHash(schainName);
-        _delayConfig[schainHash].bigTransferThreshold[token] = value;   
+        emit BigTransferThresholdIsChanged(
```
Comment (payvint) [inline proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol:291]: Also would like to add tests to check are events emitted

File: proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol (hunk 1 above)
Comment (DimaStebaev) [reply proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol]: Done.

File: proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol (hunk 2 above)
Comment (DimaStebaev) [reply proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol:291]: Updated.

File: proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol [hunk 3]
```diff
@@ -115,6 +115,34 @@ contract DepositBoxERC20 is DepositBox, IDepositBoxERC20 {
      */
     event TransferSkipped(uint256 id);
 
+    /**
+     * @dev Emitted when big transfer threshold is changed
+     */
+    event BigTransferThresholdIsChanged(
+        bytes32 indexed schainHash,
+        address token,
```
Comment (payvint) [inline proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol]: Can you make this parameter indexed too?

**Review by sergiy-skalelabs — APPROVED** [review]

File: proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol (hunk 3 above)
Comment (DimaStebaev) [reply proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol]: OK.

**Review by payvint — APPROVED** [review]

**Pull request merged**